forui/lib/src/widgets/avatar/avatar.dart

```diff
@@ -5,87 +5,69 @@ import 'package:meta/meta.dart';
 
 import 'package:forui/forui.dart';
 
+part 'avatar_content.dart';
+
 /// An image element with a fallback for representing the user.
 ///
+/// use image property to provide a profile image displayed within the circle.
 /// Typically used with a user's profile image. If the image fails to load,
-/// [placeholderBuilder] is used instead, which usually displays the user's initials.
+/// the placeholderBuilder property is used instead, which usually displays the user's initials.
 ///
-/// If the user's profile has no image, use [placeholderBuilder] to provide
+/// If the user's profile has no image, use the placeholderBuilder property to provide
 /// the initials using a [Text] widget styled with [FAvatarStyle.backgroundColor].
 class FAvatar extends StatelessWidget {
   /// The style. Defaults to [FThemeData.avatarStyle].
   final FAvatarStyle? style;
 
-  /// The profile image displayed within the circle.
-  ///
-  /// If the user's initials are used, use [placeholderBuilder] instead.
-  final ImageProvider image;
-
   /// The circle's size.
   final double size;
 
-  /// The fallback widget displayed if [image] fails to load.
+  /// The fallback widget displayed if image parameter fails to load.
   ///
   /// Typically used to display the user's initials using a [Text] widget
   /// styled with [FAvatarStyle.backgroundColor].
   ///
-  /// Use [image] to display an image; use [placeholderBuilder] for initials.
-  final Widget Function(BuildContext)? placeholderBuilder;
+  /// Use image parameter to display an image; use [placeholder] for initials.
+  final Widget placeholder;
 
   /// Creates an [FAvatar].
-  const FAvatar({
-    required this.image,
+  FAvatar({
+    required ImageProvider image,
     this.style,
     this.size = 40.0,
-    this.placeholderBuilder,
+    Widget? placeholder,
```

Comment:
Looking at it again, is `fallback` a better name? It's not really a placeholder after all.

Shadcn uses the term fallback as well.
https://ui.shadcn.com/docs/components/avatar